pages/programs/index.tsx

```diff
@@ -104,22 +107,33 @@ const pastPrograms: ProgramListingProps[] = [
 ];
 
 const Programs: React.FC = () => {
+  const createProgramModalManagement = useDisclosure();
+
   return (
-    <Container maxW={'container.lg'} mt="50px" mb="100px" fontSize="24px">
-      <Flex flexDirection="column" align="flex-start">
-        <div data-cy="programs-heading">
-          <OSLHeader header="October 2022 Online Events" style={{ marginBottom: '30px' }} />
-        </div>
-        <ProgramsList programListings={programs} />
-        <div data-cy="past-programs-heading">
-          <OSLHeader
-            header="Past Online Events"
-            style={{ marginTop: '60px', marginBottom: '30px' }}
-          />
-        </div>
-        <ProgramsList programListings={pastPrograms} />
-      </Flex>
-    </Container>
+    <>
+      <CreateProgramModal modalManagement={createProgramModalManagement} />
```

Comment:
Did we want to adjust the header so our implementation matches the Figma file?

Figma:
![image](https://user-images.githubusercontent.com/71473137/200030147-12202abb-3a5b-4611-b381-2c360535d59f.png)

Current Implementation:
![image](https://user-images.githubusercontent.com/71473137/200029965-ecae9153-e33a-49d0-9e53-9b24e0756761.png)

Reply:
I'm ok with changing it, don't really have a preference. It's just like that cuz the header was recently changed to not have the line go all the way to the end

Reply:
i think Al fixed the october events so that the line extends all the way to the end bc of Jamie's PR comment about that, so can we stay consistent :3